app/src/main/res/layout/solution_summary.xml

```diff
@@ -31,6 +31,7 @@
 
       <TextView
         android:id="@+id/solution_title"
+        style="@style/TextViewStart"
```

Comment:
Should this be `TextViewCenter`

Reply:
It is similar to hint_title text is aligned at the start.